packages/attestation-service/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@celo/attestation-service",
+  "version": "0.1.1",
+  "description": "To issue attestations for Celo'sidentity protocol",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/celo-org/celo-monorepo/tree/master/packages/attestation-service",
+  "repository": "https://github.com/celo-org/celo-monorepo/tree/master/packages/attestation-service",
+  "keywords": [
+    "celo",
+    "blockchain"
+  ],
+  "scripts": {
+    "build": "tsc -b .",
+    "clean": "tsc -b . --clean",
+    "clean:all": "yarn clean && rm -rf src/generated",
+    "build:gen": "yarn --cwd ../protocol build",
+    "prepublishOnly": "yarn build:gen && yarn build",
+    "start": "TS_NODE_FILES=true ts-node src/server.ts",
+    "db:create:dev": "mkdir -p db && touch db/dev.db",
+    "db:migrate": "sequelize db:migrate",
+    "db:migrate:dev": "sequelize db:migrate",
+    "dev": "CONFIG=config/.env.development nodemon",
+    "lint": "tslint -c tslint.json --project ."
+  },
+  "dependencies": {
+    "@celo/contractkit": "0.1.1",
+    "@celo/utils": "^0.0.6-beta5",
+    "@types/debug": "^4.1.5",
+    "bignumber.js": "^7.2.0",
+    "body-parser": "1.19.0",
+    "debug": "^4.1.1",
+    "dotenv": "8.0.0",
+    "eth-lib": "^0.2.8",
+    "nexmo": "2.4.2",
+    "web3": "1.0.0-beta.37",
+    "web3-core-helpers": "1.0.0-beta.37",
```

Comment:
not sure these web3-* packages need to be explicitly included, I think these are part of web3